velox/functions/sparksql/String.cpp

```diff
@@ -153,4 +155,110 @@ void encodeDigestToBase16(uint8_t* output, int digestSize) {
   }
 }
 
+class InitCapFunction : public exec::VectorFunction {
```

Comment:
Can this function be implemented as simple function?

Reply:
Yeah I had this question myself actually, other string functions like lowercase and uppercase are implemented as vectors so I thought I'd do the same here?

Reply:
I believe other string functions have been written before Simple Function API became powerful enough.

Reply:
Should I refactor then?

Reply:
Yes, please.